### ISSUE TITLE
test87 failing on Arch/Manjaro after update to Python 3.7

### ISSUE THREAD

**Issue by jonathonf** [vim/vim#3294]
This may not be a Vim issue but I'll report here in case it is (also for anyone else who comes looking). Arch (and Manjaro) just had a decent sized toolchain update (GCC, Python, _and_ Perl). Since this set of updates, Vim fails to compile due to test 87:

```
$ diff test87.failed test87.ok
704c704
< import failing_import:(<class 'ImportError'>, ImportError())
---
> import failing_import:(<class 'ImportError'>, ImportError('No module named failing_import',))
```

I don't see any change to the specific error format:

```
$ python
Python 3.7.0 (default, Jul 15 2018, 10:44:58) 
[GCC 8.1.1 20180531] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import failing_import
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'failing_import'
```
```
$ python3.6
Python 3.6.5 (default, May  3 2018, 10:08:28) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import failing_import
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'failing_import'
```

Possibly-related package version updates:

```
[2018-08-06 17:09] [ALPM] upgraded glibc (2.27-3 -> 2.28-1)
[2018-08-06 17:09] [ALPM] upgraded perl (5.26.2-2 -> 5.28.0-1)
[2018-08-06 17:09] [ALPM] upgraded python (3.6.6-1 -> 3.7.0-3)
[2018-08-06 17:09] [ALPM] upgraded binutils (2.30-5 -> 2.31.1-1)
[2018-08-06 17:09] [ALPM] upgraded gcc (8.1.1+20180531-1 -> 8.2.0-2)
```

Possibly-related previous change for Python 3.6: https://github.com/vim/vim/commit/f411a3c5aff644ec7a999be83ef22adf3e1132dd#diff-a4413761ae45be1ca48f5bd7b6a5b3d1

**Comment by chrisbra** [comment]
I think this is duplicate of #3162, which was just fixed in commit 447bd5a346b5c4b

**Comment by jonathonf** [comment]
Confirmed, fixed.
